modules/cert-manager-enable-operand-log-level.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-log-levels.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-operand-log-level_{context}"]
+= Setting a log level for cert-manager
+
+You can set a log level for cert-manager to determine the verbosity of log messages.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* The {cert-manager-operator} 1.11.1 or later is installed.
```

Comment:
Just a suggestion that you can consider: Is it possible to make this active voice?
E.g.
```suggestion
* You have installed the {cert-manager-operator} 1.11.1 or later.
```
If not, that's fine.